iam/sc-enduser-iam.yaml

```diff
@@ -34,7 +34,10 @@ Resources:
           - Effect: Allow
             Principal:
               AWS: !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+              # Synapse OID identity provider
+              Federated: !Sub 'arn:aws:iam::${AWS::AccountId}:oidc-provider/repo-prod.prod.sagebase.org/auth/v1'
```

Comment:
yes, that ARN looks right to me.  

(There's a small typo in the comment above:  "OID" should be "OIDC".)